pkg/config/analysis/analyzers/testdata/deployment-multi-service.yaml

```diff
@@ -4,7 +4,7 @@ metadata:
   name: bookinfo
   labels:
     istio-injection: "enabled"
-spec: {}
+spec: { }
```

Comment:
is this intentional? I don't see similar format in other files in the testate dir

Reply:
This is made by mistake with the formatter, sorry for the confusion